docs/api/environment-variables.md

```diff
@@ -51,6 +51,14 @@ Unsupported options are:
 --http-parser
 ```
 
+### `NODE_EXTRA_CA_CERTS`
+
+See [Node.js cli documentation](https://github.com/nodejs/node/blob/main/doc/api/cli.md#node_extra_ca_certsfile) for details.
+
+```sh
+export NODE_EXTRA_CA_CERTS=/path/to/cert.pem 
+```
```

Comment:
We should possibly mention the fuses here and in the NODE_OPTIONS section above?

Reply:
done in 94303959a3